src/invocation/ClusterService.ts

```diff
@@ -134,10 +140,31 @@ export class ClusterService implements Cluster {
         }
     }
 
-    reset(): void {
+    onClusterConnect(): void {
+        this.logger.debug('ClusterService', 'Resetting the member list version.');
```

Comment:
java uses fine logging here which should be the most detailed logging. So, can we use `.trace` here?

Reply:
Fixed